expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -161,19 +161,6 @@ void shouldReturnIndexDocumentWithValidReferenceData() throws Exception {
         assertTrue(publisher.isNotEmpty());
     }
 
-    @Test
-    void shouldReturnIndexDocumentWithValidContributorAffiliationCountryCode() throws Exception {
-        var publication = randomPublication();
-        FakeUriResponse.setupFakeForType(publication, fakeUriRetriever, resourceService);
-
-        var indexDocument = fromPublication(fakeUriRetriever, resourceService, publication);
-        var framedResultNode = indexDocument.asJsonNode();
-
-        var actualCountryCode = framedResultNode.at("/entityDescription/contributors/1/affiliations/0/countryCode")
-                                    .textValue();
-        assertThat(actualCountryCode, is(not(nullValue())));
-    }
-
```

Comment:
This test always fail, @brinxmat knows why.

Reply:
Have fixed